tests/tests/test_log.rs

```diff
@@ -274,34 +286,33 @@ fn log_iterator() {
         .io_buf_size(1000)
         .build();
     let log = Log::start_raw_log(config.clone()).unwrap();
-    let (first_lsn, _) = log.write(b"".to_vec()).unwrap();
-    log.write(b"1".to_vec());
-    log.write(b"22".to_vec());
-    log.write(b"333".to_vec());
+    let (first_lsn, _) = log.write(b"").unwrap();
+    log.write(b"1");
+    log.write(b"22");
+    log.write(b"333");
 
     // stick an abort in the middle, which should not be
     // returned
     {
-        let res =
-            log.reserve(b"never_gonna_hit_disk".to_vec()).unwrap();
+        let res = log.reserve(b"never_gonna_hit_disk").unwrap();
         res.abort().unwrap();
     }
 
-    log.write(b"4444".to_vec());
-    let (last_lsn, _) = log.write(b"55555".to_vec()).unwrap();
+    log.write(b"4444");
```

Comment:
Just for information: [the primitive `str` type implement `AsRef<[u8]>`](https://doc.rust-lang.org/std/primitive.str.html#impl-AsRef%3C%5Bu8%5D%3E), it means that it is possible to directly call `log.write("4444")`.